pkg/kubelet/kubelet.go

```diff
@@ -368,21 +364,19 @@ func NewMainKubelet(
 	}
 	klet.containerManager = containerManager
 
-	go util.Until(klet.syncNetworkStatus, 30*time.Second, util.NeverStop)
-	if klet.kubeClient != nil {
-		// Start syncing node status immediately, this may set up things the runtime needs to run.
```

Comment:
I did not notice any cross dependencies between this logic and the rest of the function. The node status will still be synced before the sync loop starts.